src/tools/rbd/action/Snap.cc

```diff
@@ -237,19 +237,15 @@ int do_purge_snaps(librbd::Image& image, bool no_progress)
 int do_protect_snap(librbd::Image& image, const char *snapname)
 {
   int r = image.snap_protect(snapname);
-  if (r < 0)
-    return r;
 
-  return 0;
+  return r < 0 ? r : 0;
```

Comment:
This is actually a conscience choice throughout RBD since it makes it easier to set gdb breakpoints. Switching just these two return statements to ternary operators seems more like "change for the sake for change"

Reply:
thans a lot.  but I just feel it is more concise.

Reply:
That's the problem w/ style -- two people can always have different ideas. That's why we don't just change things for the sake of changing them.

Reply:
I see. thanks